src/components/Banner/Banner.tsx

```diff
@@ -1,17 +1,25 @@
 import styles from './Banner.module.css';
 import type { LayoutText } from 'custom-types';
-import _layoutText from '../../../config/devfile-layout-text.json';
+import _layoutText from '@info/layout-text.json';
+import { Wave } from '@src/components';
+import { getLayoutText } from '@src/util/client';
 import { Text, TextContent, TextVariants } from '@patternfly/react-core';
 import { useRouter } from 'next/router';
-import { Wave } from '@src/components';
-
-const layoutText = _layoutText as LayoutText;
+import { useState, useEffect } from 'react';
 
 export const Banner: React.FC = () => {
   const router = useRouter();
+  const [layoutText, setLayoutText] = useState<LayoutText>(_layoutText);
+
+  useEffect(() => {
+    getLayoutText().then((layoutTextRes) => {
+      setLayoutText(() => layoutTextRes);
+    });
+  }, []);
+
   return (
     <>
-      {router.asPath === '/' ? (
+      {router.asPath === '/' && !process.env.DEVFILE_BANNER ? (
```

Comment:
![Screen Shot 2021-10-15 at 5 19 38 PM](https://user-images.githubusercontent.com/31771087/137554840-738e819d-798b-494c-9ec0-82021c36e00d.png)

I see a lot of white space